src/routes/boot.ts

```diff
@@ -950,4 +950,14 @@ export default async function (fastify: FastifyInstance): Promise<void> {
       (req) => req.cookies[cookies.funnel.key],
     ),
   );
+
+  // Used to get the boot data for onboarding funnels
+  fastify.get(
+    '/onboarding',
+    funnelHandler(
```

Comment:
Can you explain why different endpoint for onboarding, just out of loop?

Reply:
and for example will we have more of these endpoints in the future with different cookies, maybe generic one with `:id` param would be better?

Reply:
We are making the current /onboarding behave the same way as the Funnel does. Config based.

Reply:
Yes basically it is the same as the funnel BUT we need:
- different session id to retrieve (so we need a new cookie key)
- set a new GB feature key in order to retrieve the actual default version 

We could add a generic :id but we don't want to handle other cases then the `/onboarding` & `/funnel` for now. 
I can create an object ( similar to this `{ [endpointId]: { cookieKey, featureKey } }` ) and loop through the properties if we want to have a clearer overview of it. Wdyt?

Reply:
Updated ✔️